Python/jit.c

```diff
@@ -113,6 +114,21 @@ mark_executable(unsigned char *memory, size_t size)
 
 // JIT compiler stuff: /////////////////////////////////////////////////////////
 
+#define SYMBOL_MASK_WORDS 4
+
+typedef uint32_t SymbolMask[SYMBOL_MASK_WORDS];
```

Comment:
Let's make these 64 bits (and adjust the writer accordingly). Also, lets stick with `snake_case` for stuff local to this file:

```suggestion
#define SYMBOL_MASK_WORDS 2

typedef uint64_t symbol_mask[SYMBOL_MASK_WORDS];
```

Reply:
The reason there are 32 bit is because we can use _Py_popcount32 against the uint32_t data type.

Reply:
Also the 32bit gives us better granularity of the size of the mask. Currently the symbol_mask is 128bit and the next step would be  160bit whilst if we use 64 bit, it would be 192 with the majority of the new 64 bits empty anyway.

Reply:
Thanks, 32-bits makes sense. I missed that there's no 64-bit variant of the popcount function.